src/locale/properties/register/register.xx.properties

```diff
@@ -23,7 +23,6 @@ register.additionalEmailInfo=X
 register.consentTo=X
 register.privacyPolicy=X
 register.and=X
-register.termsAndConditions=X
```

Comment:
Property `register.terms` is missing in all testing languages

Reply:
that one was removed, now `register.terms` is been use instead